src/core_atmosphere/physics/mpas_atmphys_driver_pbl.F

```diff
@@ -930,6 +1057,184 @@ subroutine driver_pbl(itimestep,configs,mesh,sfc_input,diag_physics,tend_physics
+! call mpas_dmpar_min_real(domain % dminfo, minval(ht_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(ht_p), scalar_max)
+!call mpas_log_write('MIN/MAX ht_p = $r, $r', realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(qsfc_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(qsfc_p), scalar_max)
+!call mpas_log_write('MIN/MAX qsfc_p = $r, $r', realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(thz0_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(thz0_p), scalar_max)
+!call mpas_log_write('MIN/MAX thz0_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(qz0_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(qz0_p), scalar_max)
+!call mpas_log_write('MIN/MAX qz0_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(uz0_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(uz0_p), scalar_max)
+!call mpas_log_write('MIN/MAX uz0_p = $r, $r', realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(vz0_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(vz0_p), scalar_max)
+!call mpas_log_write('MIN/MAX vz0_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(tkepbl_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(tkepbl_p), scalar_max)
+!call mpas_log_write('MIN/MAX tkepbl_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(kzh_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(kzh_p), scalar_max)
+!call mpas_log_write('MIN/MAX kzh_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(ust_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(ust_p), scalar_max)
+!call mpas_log_write('MIN/MAX ust_p = $r, $r', realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(znt_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(znt_p), scalar_max)
+!call mpas_log_write('MIN/MAX znt_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(ct_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(ct_p), scalar_max)
+!call mpas_log_write('MIN/MAX ct_p = $r, $r',realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(akhs_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(akhs_p), scalar_max)
+!call mpas_log_write('MIN/MAX akhs_p = $r, $r', realArgs=(/scalar_min,scalar_max/)
+!
+! call mpas_dmpar_min_real(domain % dminfo, minval(akms_p), scalar_min)
+! call mpas_dmpar_min_real(domain % dminfo, maxval(akms_p), scalar_max)
+!call mpas_log_write('MIN/MAX akms_p = $r, $r', realArgs=(/scalar_min,scalar_max/)
+
```

Comment:
looks like a lot of leftover debugging code, but I suppose it doesn't hurt to keep it for now.